api/controllers/users.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+const bcrypt = require('bcrypt');
+const random = require('../../utils/random');
```

Comment:
why not use `uuid` package? 😃

Reply:
to generate random strings?
As the id field should be 16 char I think we don't need another lib, but im ok with it.

Reply:
it's just a suggestion. we can move forward with that